dart_sp/11.Dart_medio/2_La_terminal.md

```diff
@@ -44,9 +44,9 @@ Como para terminar esta capaz no tan breve introducción, les dejo un listado de
 - `kill PID`: mata un programa utilizando su PID. Por ejemplo: `kill 4320`.
 - `ls <opciones> <ficheros>`: lista el contenido de un directorio.
 - `cp <opciones> <archivo> <ruta>`: copia un archivo o directorio en el directorio especificado.
-- `mv <opciones> <archivo> <ruta>`: mueve un archivo o directorio a un archivo o directorio. Se utiliza también para renombrar; el origen y destino es un archivo.
+- `mv <opciones> <archivo> <ruta>`: se utiliza  para mover archivos y directorios de un directorio a otro, o para renombrar un archivo o directorio.
 - `cat <opciones> <archivos>`: muestra por pantalla el contenido de los archivos por la salida estándar.
 -`find <ruta(s)> <condición(es) de búsqueda> <acciones>`: busca archivos recorriendo árboles de directorios especificados por las rutas, evaluando de izquierda a derecha las condiciones de búsqueda. Además se pueden especificar acciones sobre los resultados obtenidos.
 -`pwd <opciones>`: Muestra el nombre del directorio en el que uno se encuentra situado (_print working directory_).
 
-Espero que tengan suficiente para meter sus manitas a explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️
+Podrán ver que muchos de estos comandos no funcionan en Windows, pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️
```

Comment:
Si te parece, yo buscaría ya que los estás conociendo mientras los hacés, cuáles son los que faltan e ir sumándolos gradual y puntualmente.

Reply:
Seems good, but as a future task. I don't think it's really necessary for the course or for me right now. If you agree, we leave this for later.
Maybe one more modification: from this "Podrán ver que muchos de estos comandos no funcionan en Windows, pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️"
to this "Si estan trabajando en Windows podrán ver que muchos de estos comandos no funcionan (algunos tienen otro nombre), pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️"